frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -107,77 +94,167 @@ const DashboardInfoSidebarBody = ({
+                  descriptionError={descriptionError}
+                  setDescriptionError={setDescriptionError}
+                  canWrite={canWrite}
+                  setPage={setPage}
+                  showCaching={showCaching}
+                />
+              </Tabs.Panel>
+              <Tabs.Panel value={Tab.History}>
+                <HistoryTab
+                  canWrite={canWrite}
+                  revisions={revisions}
+                  currentUser={currentUser}
+                />
+              </Tabs.Panel>
+            </SidesheetTabPanelContainer>
+          </Tabs>
+        </Sidesheet>
+      </ErrorBoundary>
+    </div>
+  );
+}
+
+const OverviewTab = ({
+  dashboard,
+  handleDescriptionChange,
+  handleDescriptionBlur,
+  descriptionError,
+  setDescriptionError,
+  canWrite,
+  setPage,
+  showCaching,
+}: {
+  dashboard: Dashboard;
+  handleDescriptionChange: (description: string) => void;
+  handleDescriptionBlur: (event: FocusEvent<HTMLTextAreaElement>) => void;
+  descriptionError: string | null;
+  setDescriptionError: (error: string | null) => void;
+  canWrite: boolean;
+  setPage: (
+    page: "default" | "caching" | SetStateAction<"default" | "caching">,
+  ) => void;
+  showCaching: boolean;
+}) => {
+  const isCacheable = isDashboardCacheable(dashboard);
+  const autoApplyFilterToggleId = useUniqueId();
+  const dispatch = useDispatch();
   const handleToggleAutoApplyFilters = useCallback(
     (isAutoApplyingFilters: boolean) => {
       dispatch(toggleAutoApplyFilters(isAutoApplyingFilters));
     },
     [dispatch],
   );
 
-  const autoApplyFilterToggleId = useUniqueId();
-  const canWrite = dashboard.can_write && !dashboard.archived;
-  const isCacheable = isDashboardCacheable(dashboard);
-
-  const showCaching = canWrite && PLUGIN_CACHING.isGranularCachingEnabled();
-
   return (
-    <>
-      <ContentSection>
-        <DescriptionHeader>{t`About`}</DescriptionHeader>
-        <EditableDescription
-          initialValue={dashboard.description}
-          isDisabled={!canWrite}
-          onChange={handleDescriptionChange}
-          onFocus={() => setDescriptionError("")}
-          onBlur={handleDescriptionBlur}
-          isOptional
-          isMultiline
-          isMarkdown
-          hasError={!!descriptionError}
-          placeholder={t`Add description`}
-          key={`dashboard-description-${dashboard.description}`}
-          style={{ fontSize: ".875rem" }}
-        />
+    <Stack spacing="lg">
+      <SidesheetCard title={t`Description`} pb="md">
+        <div className={DashboardInfoSidebarS.EditableTextContainer}>
+          <EditableText
+            initialValue={dashboard.description}
+            isDisabled={!canWrite}
+            onChange={handleDescriptionChange}
+            onFocus={() => setDescriptionError("")}
+            onBlur={handleDescriptionBlur}
+            isOptional
+            isMultiline
+            isMarkdown
+            placeholder={t`Add description`}
```

Comment:
cc: @vbenedetti

I might be wrong, of course, but since it already says "Description" in the header, I wonder if there's a more thoughtful placeholder we could use. Perhaps "Describe this dashboard"?